spk/syncthing/src/service-setup.sh

```diff
@@ -1,23 +1,25 @@
 # syncthing service definition
 SYNCTHING="${SYNOPKG_PKGDEST}/bin/syncthing"
-SERVICE_COMMAND="${SYNCTHING} serve --home=${SYNOPKG_PKGVAR}"
+# define folder for configuration (config, keys, database, logs)
+SYNCTHING_CONFIG="--config=${SYNOPKG_PKGVAR} --data=${SYNOPKG_PKGVAR}"
+SERVICE_COMMAND="${SYNCTHING} serve ${SYNCTHING_CONFIG}"
 SVC_BACKGROUND=y
 SVC_WRITE_PID=y
 
 GROUP="sc-syncthing"
 
-# Required to run any syncthing command: set $HOME environment variable
-HOME=${SYNOPKG_PKGVAR}
-export HOME
-
 # include next gen gui
 export STGUIASSETS=${SYNOPKG_PKGDEST}/gui
 
 set_credentials() {
     if [ -n "${wizard_username}" -a -n "${wizard_password}" ]; then
+        echo "create user ${wizard_username} for syncthing web gui access"
```

Comment:
I find this message a bit misleading.  Normally when "creating a user", we are talking about a DSM system level user account.
```suggestion
        echo "set user name ${wizard_username} for syncthing Web GUI access"
```